Dockerfile.bundle

```diff
@@ -7,6 +7,9 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=external-dns-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=stable-v1.3,stable-v1
 LABEL operators.operatorframework.io.bundle.channel.default.v1=stable-v1
+LABEL io.openshift.build.commit.id=${SHORT-COMMIT}
+LABEL io.openshift.build.source-location=${GET_URL}
+LABEL io.openshift.build.commit.url=${GET_URL}/commit/${SHORT-COMMIT}
```

Comment:
Let's keep the source bundle Dockerfile as is for the moment. Like it was on CPaaS - dynamic metadata was added at build time only.

```suggestion
```